tests/symbolicator/test_unreal_full.py

```diff
@@ -80,21 +80,33 @@ def unreal_crash_test_impl(self, filename):
             with open(filename, "rb") as f:
                 event = self.post_and_retrieve_unreal(f.read())
 
-        self.insta_snapshot(
-            {
-                "contexts": event.data.get("contexts"),
-                "exception": {
-                    "values": [
-                        normalize_exception(x)
-                        for x in get_path(event.data, "exception", "values") or ()
-                    ]
+        def make_snapshot(subname=None):
+            self.insta_snapshot(
+                {
+                    "contexts": event.data.get("contexts"),
+                    "exception": {
+                        "values": [
+                            normalize_exception(x)
+                            for x in get_path(event.data, "exception", "values") or ()
+                        ]
+                    },
+                    "stacktrace": event.data.get("stacktrace"),
+                    "threads": event.data.get("threads"),
+                    "extra": event.data.get("extra"),
+                    "sdk": event.data.get("sdk"),
                 },
-                "stacktrace": event.data.get("stacktrace"),
-                "threads": event.data.get("threads"),
-                "extra": event.data.get("extra"),
-                "sdk": event.data.get("sdk"),
-            }
-        )
+                subname=subname,
+            )
+
+        # We do snapshot tests against two different snapshots here, to be
+        # forward compatible with the upcoming changes from
+        # https://github.com/getsentry/symbolicator/pull/794
+        # This is temporary and will go away once symbolicator is merged and
+        # the docker image used in sentrys own tests is updated.
+        try:
+            make_snapshot()
+        except BaseException:
```

Comment:
this should be `except Exception:` at least (or a more specific `AssertionError` / `ValueError` if possible) -- `BaseException` catches `SystemExit` / `KeyboardInterrupt` which *should* terminate the process if raised

I did a video on the topic (well, more specifically blank `except:` which is equivalent to `except BaseException:`) a while back: https://youtu.be/i0GOLe-F27Q

Reply:
yep yep -- that's why I commented+approved.  not critical to fix in this PR